examples/givens_rotations.ipynb

```diff
@@ -0,0 +1,293 @@
+{
```

Comment:
Making an ipython notebook for this is really going above and beyond. Great work! If you do want to include this though I will ask that you add a very primitive test that checks to make sure that the ipython notebook runs without throwing any errors (no need to test further). The reason we do this is so that examples in ipython notebooks don't get silently broken by later updates (which used to happen to use a lot). I implemented such a test for our demo in tests/_example_test.py

You can add the test for your ipython notebook to that file as well. Just copy the same format. Note that some aspects of this way of testing are a little hacky. The worst part is that you will need to add those lines with "chdir" at the top of the ipython notebook (see what I did in the demo). If you have a better idea for how to get these tests to pass on travis though, do let us know!